source/documentation/values/calculations.html.md.erb

```diff
@@ -258,3 +258,117 @@ combined at build time, so `min(12px % 10, 10%)` will throw an error.
     padding-right: 20px;
   }
 <% end %>
+
+### [`round()`]
+<% impl_status dart: "1.64.0", libsass: false, ruby: false, feature: "Special function syntax" do %>
+  LibSass, Ruby Sass, and versions of Dart Sass prior to 1.64.0 *always* parse
+  `round()` a as Sass function. To create a plain CSS calculation function
+  call for those implementations, you can write something like
+  `unquote("round(#{$number})")` instead.
+
+  [special functions]: ../syntax/special-functions
+<% end %>
+
+The `round(<strategy>, value, step)` function accepts an optionl rounding strategy, 
+a value to be rounded and a rounding interval `step`. 
+`strategy` should be `nearest`, `up`, `down`, or `to-zero`.
+`round(value)` is also supported for backwards compatibility.
+
+<% example do %>
+  $number: 12.5px;
+  $step: 15px;
+
+  .post {
+    // Since these round() calls are valid calculation expressions, they're
+    // parsed as calculations.
+    padding-left: round(nearest, $number, $step);
+    padding-right: round($number + 10px);
+    padding-bottom: round($number + 10px, $step + 10%);
+  }
+
+  ===
+  $number: 12.5px;
+  $step: 15px;
+
+  .post
+    // Since these round() calls are valid calculation expressions, they're
+    // parsed as calculations.
+    padding-left: round(nearest, $number, $step)
+    padding-right: round($number + 10px)
+    padding-bottom: round($number + 10px, $step + 10%)
+
+  ===
+  .post {
+    padding-left: 15px;
+    padding-right: 23px;
+    padding-bottom: round(22.5px, 15px + 10%);
+  }
+<% end %>
+
+### [`abs()`]
+<% impl_status dart: "1.64.0", libsass: false, ruby: false, feature: "Special function syntax" do %>
+  LibSass, Ruby Sass, and versions of Dart Sass prior to 1.64.0 *always* parse
+  `abs()` a as Sass function. To create a plain CSS calculation function
+  call for those implementations, you can write something like
+  `unquote("abs(#{$number})")` instead.
+
+  [special functions]: ../syntax/special-functions
+<% end %>
+
+Given that CSS resolves percentages before calculations, 
+the global `abs(value)` function compatibiliy with % unit parameters is deprecated.
+In the future, this will emit a CSS abs() function to be resolved by the browser.
+
+<% example do %>
+  .post {
+    padding-left: abs(-10px);
+    padding-right: math.abs(-7.5%);
+  }
+
+  ===
+  .post
+    padding-left: abs(-10px)
+    padding-right: math.abs(-7.5%)
+
+  ===
+  .post {
+    padding-left: 10px;
+    padding-right: 7.5%;
+  }
+<% end %>
+
+### Other math functions
+<% impl_status dart: "1.64.0", libsass: false, ruby: false, feature: "Special function syntax" do %>
+  LibSass, Ruby Sass, and versions of Dart Sass prior to 1.64.0 *always* parse
+  the following functions as Sass functions. To create a plain CSS calculation function
+  call for those implementations, you can write something like
+  `unquote("sqrt(#{$number})")` instead.
```

Comment:
This impl status should go immediately below the one for first-class calc in 1.40.0 and read similarly. Except for `min`, `max`, `round`, and `abs`, the other functions were never Sass global functions, so there's no backwards compatibility to maintain. Doing this unquote trick is unnecessary, since Sass just defaults to emitting a CSS function if there's no Sass function with that name. 1.64.0 is just the first version where these will simplify during Sass compilation.

Reply:
Sounds good, just added a `impl_status` under the introduction title